frontend/src/reducers/account.js

```diff
@@ -1,10 +1,24 @@
+import * as Cookies from "js-cookie";
+import {browserHistory} from 'react-router';
+
 export default function accounts(state = [], action) {
     switch (action.type) {
         case 'ADD_ACCOUNT_SUCCESS':
-            return state
+            browserHistory.push('/home');
+            return Object.assign({}, state, {
+                error: ''
+            });
+        case 'ADD_ACCOUNT_FAIL':
+            return Object.assign({}, state, {
+                error: action.error.response.data.Message
+            });
         case 'LOAD_ACCOUNTS_SUCCESS':
-            let accountList = action.payload.data
-            return accountList;
+            let accountsList = action.payload.data;
+            return accountsList;
+        case 'LOGOUT_SUCCESS':
+            const nameCookie = "SID";
+            Cookies.remove(nameCookie);
```

Comment:
You can just call Cookies.remove("SID") 

It's strange that login is not handled here, if it's here, then the name of the cookie could be extracted as constant.